src/redux/modules/surahs.js

```diff
@@ -34,6 +36,11 @@ export default function reducer(state = initialState, action = {}) {
     case LOAD_FAIL:
       console.log(action);
       return state;
+    case TOGGLE_SURAH_INFO:
```

Comment:
i dont get it, how does this work? this is surahs.js and setOptionDispatch runs in options.js